app.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
I want you to setup eslint and use the Airbnb style guide. It will make your code much cleaner.